travis/install-wdl.sh

```diff
@@ -17,11 +17,11 @@ cached_fetch_jar_from_github () {
 	ln -s $CACHE_DIR/$_jar_fname $_tool_name.jar
 }
 
-cached_fetch_jar_from_github broadinstitute cromwell womtool 33.1
-cached_fetch_jar_from_github broadinstitute cromwell cromwell 33.1
-cached_fetch_jar_from_github dnanexus dxWDL dxWDL 0.72
+cached_fetch_jar_from_github broadinstitute cromwell womtool 47
+cached_fetch_jar_from_github broadinstitute cromwell cromwell 47
+cached_fetch_jar_from_github dnanexus dxWDL dxWDL v1.33
 
-TGZ=dx-toolkit-v0.285.0-ubuntu-16.04-amd64.tar.gz
+TGZ=dx-toolkit-v0.288.0-ubuntu-16.04-amd64.tar.gz
```

Comment:
Looks like v0.289 is available:
https://dnanexus-sdk.s3.amazonaws.com/dx-toolkit-v0.289.0-ubuntu-16.04-amd64.tar.gz

Reply:
well now that's annoying... it wasn't listed on https://documentation.dnanexus.com/downloads